src/pages/CvsPage/components/CvInfo/CvInfo.tsx

```diff
@@ -37,63 +40,77 @@ export const CvInfo = ({ cvId }: CvInfoProps) => {
     // show projects table
   };
 
+  const showPreview = () => {
+    setIsPatternsVisible(true);
+  };
+
+  const hidePreview = () => {
+    setIsPatternsVisible(false);
+  };
+
   return (
-    <form onSubmit={handleSubmit(onSubmit)}>
-      <InfoFormWrapper>
-        <Fieldset
-          control={control}
-          required="Please, specify the field"
-          label="Email"
-          name="email"
-        />
-        <Fieldset
-          control={control}
-          required="Please, specify the field"
-          label="CV Name"
-          name="lastName"
-        />
-        <Fieldset
-          control={control}
-          required="Please, specify the field"
-          label="Skills"
-          name="skills"
-        />
-        <Fieldset
-          control={control}
-          required="Please, specify the field"
-          label="Specialization"
-          name="specialization"
-        />
-        <Fieldset
-          control={control}
-          required="Please, specify the field"
-          label="Department"
-          name="department"
-        />
-      </InfoFormWrapper>
+    <>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <InfoFormWrapper>
+          <Fieldset
+            control={control}
+            required="Please, specify the field"
+            label="Email"
+            name="email"
+          />
+          <Fieldset
+            control={control}
+            required="Please, specify the field"
+            label="CV Name"
+            name="lastName"
+          />
+          <Fieldset
+            control={control}
+            required="Please, specify the field"
+            label="Skills"
+            name="skills"
+          />
+          <Fieldset
+            control={control}
+            required="Please, specify the field"
+            label="Specialization"
+            name="specialization"
+          />
+          <Fieldset
+            control={control}
+            required="Please, specify the field"
+            label="Department"
+            name="department"
+          />
```

Comment:
Maybe we can use [{
required,label,name
}].map(({
required,label,name
})=><Feldset
required
label
name
control
}
 />